tests/execution_engine/test_sqlalchemy_batch_data.py

```diff
@@ -102,7 +124,7 @@ def test_instantiation_with_and_without_temp_table(sqlite_view_engine, sa):
     # testing whether schema is supported
     selectable = sa.select("*").select_from(sa.table(name="test_table", schema="main"))
     SqlAlchemyBatchData(
-        execution_engine=engine,
+        execution_engine=execution_engine,
```

Comment:
I believe `engine` on line 142 needs to be changed to `execution_engine`.

Reply:
There is no line 142 there any more...  And it says `execution_engine` everywhere.  I know that `GitHub` is not always clear about the changes, and sometimes you one has to go to the working branch to be sure.  Happy to pair on this.  Thanks.

Reply:
Ooo!  I found one `engine` and fixed it.  Thanks.  (It was on line 139.)